support/developer/visualstudio/general/visual-studio-2022-unsupported-operating-systems.md

```diff
@@ -52,6 +52,7 @@ Visual Studio 2022 doesn't install on Windows 8.
 
 When installing or using Visual Studio 2022 on Windows 8.1, Windows Server 2012, or Windows Server 2012 R2, you may experience the following issues.
 
+- Windows 8.1, Windows Servcer 2012, and Windows Server 2012 R2 are no longer supported on Visual Studio 2022, and will not recieve future security updates. You may be in an unsecure state when installing on these operating systems. 
```

Comment:
Spelling error on Windows Servcer 2012

Reply:
@AdrianMurphy - These OS were never supported for VS2022. I'm not sure what this line means?

Reply:
Corrected/reworded